chain/network/src/peer.rs

```diff
@@ -165,6 +173,8 @@ pub struct Peer {
     last_time_received_message_update: Instant,
     /// Dynamic Prometheus metrics
     network_metrics: NetworkMetrics,
+    /// How many transactions we have received since the last block message
+    txns_since_last_block: Arc<RwLock<usize>>,
```

Comment:
Use `Arc<AtomicUsize>` instead?

Reply:
Done in https://github.com/nearprotocol/nearcore/pull/3317/commits/7ee90fbf5a49302c9deab18e832475a472f2e772